portfolio/src/main/java/com/google/sps/servlets/QuoteGenerator.java

```diff
@@ -9,10 +13,48 @@
 /** Handles requests sent to the /quote URL endpoint */
 @WebServlet("/quote")
 public class QuoteGenerator extends HttpServlet {
-
   @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>\"Life changes fast. Life changes in the instant.\" - Joan Didion</h1>");
+  public void doGet(final HttpServletRequest request, final HttpServletResponse response)
+      throws IOException {
+    // Get a random quote
+    Quote randomQuote = this.getRandomQuote();
+
+    // Parse the quote and return the json output
+    final Gson gson = new Gson();
+    final String json = gson.toJson(randomQuote);
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+  }
+
+  /**
+   * Build the quotes using the author name and quote text.
```

Comment:
nit: The recommended comment style is third person. So, "Builds a quote using the author name and quote text" or just "Builds a quote"

https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#styleguide